tests/luatest.lua

```diff
@@ -1,4 +1,4 @@
-os.execute([[.\..\Debug\flatc.exe --lua -I include_test monster_test.fbs]])
+os.execute([[.\..\Release\flatc.exe --lua -I include_test monster_test.fbs]])
```

Comment:
This is very Windows specific. Maybe better to leave this line out, and assume that the files have already been generated prior to running this test (that's how it works for most Languages).